File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarLotesOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarLotesOut : ParametroBaseOut
    {
        public List<Lote> Lotes { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarLotesOut.ts
import { Lote } from "src/app/Components/Lote/class/lote";
import { ParametroBaseOut } from "./ParametroBaseOut";

export class ListarLotesOut extends ParametroBaseOut{  

    private lotes: Lote[];
    
    constructor()
    {
        super();
        this.lotes = [];
    }

    public get Lotes(): Lote[] {
        return this.lotes;
    }
    public set Lotes(value: Lote[]) {
        this.lotes = value;
    }
}